### PR TITLE
MINOR: remove get prefix for internal IQ methods

### PR DESCRIPTION
**Pull request by mjsax**
Follow up PR to https://github.com/apache/kafka/pull/16722

### CODE REVIEW

**Review by bbejeck — APPROVED** [review]
Thanks for the cleanup @mjsax - LGTM

File: streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsMetadataState.java [hunk 1]
```diff
@@ -206,29 +206,29 @@ public synchronized Collection<StreamsMetadata> getAllMetadataForTopology(final
...
      */
-    public synchronized <K> KeyQueryMetadata getKeyQueryMetadataForKey(final String storeName,
-                                                                       final K key,
-                                                                       final Serializer<K> keySerializer) {
+    public synchronized <K> KeyQueryMetadata keyQueryMetadataForKey(final String storeName,
+                                                                    final K key,
+                                                                    final Serializer<K> keySerializer) {
         Objects.requireNonNull(keySerializer, "keySerializer can't be null");
         if (topologyMetadata.hasNamedTopologies()) {
             throw new IllegalArgumentException("Cannot invoke the getKeyQueryMetadataForKey(storeName, key, keySerializer)"
```
Comment (chia7712) [inline streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsMetadataState.java]: please update the error message also

File: streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsMetadataState.java [hunk 2]
```diff
@@ -206,29 +206,29 @@ public synchronized Collection<StreamsMetadata> getAllMetadataForTopology(final
...
+    public synchronized <K> KeyQueryMetadata keyQueryMetadataForKey(final String storeName,
+                                                                    final K key,
+                                                                    final Serializer<K> keySerializer,
+                                                                    final String topologyName) {
         Objects.requireNonNull(keySerializer, "keySerializer can't be null");
-        return getKeyQueryMetadataForKey(storeName,
+        return keyQueryMetadataForKey(storeName,
                                          key,
                                          new DefaultStreamPartitioner<>(keySerializer),
                                          topologyName);
```
Comment (chia7712) [inline streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsMetadataState.java]: please update error message (line#257) also

File: streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsMetadataState.java [hunk 3]
```diff
@@ -112,7 +112,7 @@ public Collection<StreamsMetadata> getAllMetadata() {
      * @param storeName the storeName to find metadata for
      * @return A collection of {@link StreamsMetadata} that have the provided storeName
      */
-    public synchronized Collection<StreamsMetadata> getAllMetadataForStore(final String storeName) {
+    public synchronized Collection<StreamsMetadata> allMetadataForStore(final String storeName) {
         Objects.requireNonNull(storeName, "storeName cannot be null");
         if (topologyMetadata.hasNamedTopologies()) {
             throw new IllegalArgumentException("Cannot invoke the getAllMetadataForStore(storeName) method when"
```
Comment (chia7712) [inline streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsMetadataState.java]: please update message

File: streams/src/main/java/org/apache/kafka/streams/processor/internals/Task.java [hunk 4]
```diff
@@ -246,7 +246,7 @@ default boolean commitRequested() {
 
     // IQ related methods
 
-    StateStore getStore(final String name);
+    StateStore store(final String name);
```
Comment (chia7712) [inline streams/src/main/java/org/apache/kafka/streams/processor/internals/Task.java:249]: Should we rename `StateManager#getStore(String)` to `StateManager#store(String)`

**Review by chia7712 — COMMENTED** [review]
@mjsax thanks for this cleanup. a couple of comments are left. PTAL

File: streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsMetadataState.java (hunk 1 above)
Comment (mjsax) [reply streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsMetadataState.java]: Good catch!

File: streams/src/main/java/org/apache/kafka/streams/processor/internals/Task.java (hunk 4 above)
Comment (mjsax) [reply streams/src/main/java/org/apache/kafka/streams/processor/internals/Task.java:249]: Yes, but I will do this in a follow up PR to keep the scope of individual PRs small.

**Comment by chia7712** [comment]
I open https://issues.apache.org/jira/browse/KAFKA-17420 to trace `StreamThreadTest.tearDown`, and others are known flaky.

**Review by chia7712 — APPROVED** [review]

File: streams/src/main/java/org/apache/kafka/streams/processor/internals/Task.java (hunk 4 above)
Comment (mjsax) [reply streams/src/main/java/org/apache/kafka/streams/processor/internals/Task.java:249]: Here we go: https://github.com/apache/kafka/pull/17025

**Pull request merged**